<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Genomics üß¨ | Bioinformatics Notes üß¨</title>
  <meta name="description" content="A R Bookdown üìñ Project containing my notes on Bioinformatics ü¶† and Genomics üß¨" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Genomics üß¨ | Bioinformatics Notes üß¨" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A R Bookdown üìñ Project containing my notes on Bioinformatics ü¶† and Genomics üß¨" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Genomics üß¨ | Bioinformatics Notes üß¨" />
  
  <meta name="twitter:description" content="A R Bookdown üìñ Project containing my notes on Bioinformatics ü¶† and Genomics üß¨" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="basics.html"/>
<link rel="next" href="systems-biology.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Bioinformatics Notes üß¨</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="basics.html"><a href="basics.html"><i class="fa fa-check"></i><b>2</b> Basics of R</a><ul>
<li class="chapter" data-level="2.1" data-path="basics.html"><a href="basics.html#why-r-Ô∏è"><i class="fa fa-check"></i><b>2.1</b> Why R ? ü§∑üèªÔ∏è</a></li>
<li class="chapter" data-level="2.2" data-path="basics.html"><a href="basics.html#installing-r"><i class="fa fa-check"></i><b>2.2</b> Installing R üóº</a></li>
<li class="chapter" data-level="2.3" data-path="basics.html"><a href="basics.html#basic-syntax"><i class="fa fa-check"></i><b>2.3</b> Basic Syntax</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="genomics.html"><a href="genomics.html"><i class="fa fa-check"></i><b>3</b> Genomics üß¨</a><ul>
<li class="chapter" data-level="3.1" data-path="genomics.html"><a href="genomics.html#basics-of-genomics"><i class="fa fa-check"></i><b>3.1</b> Basics of Genomics üß¨</a><ul>
<li class="chapter" data-level="3.1.1" data-path="genomics.html"><a href="genomics.html#transcription"><i class="fa fa-check"></i><b>3.1.1</b> Transcription</a></li>
<li class="chapter" data-level="3.1.2" data-path="genomics.html"><a href="genomics.html#translation"><i class="fa fa-check"></i><b>3.1.2</b> Translation</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="genomics.html"><a href="genomics.html#genetic-markers"><i class="fa fa-check"></i><b>3.2</b> Genetic Markers üñä</a><ul>
<li class="chapter" data-level="3.2.1" data-path="genomics.html"><a href="genomics.html#what-are-markers-Ô∏è"><i class="fa fa-check"></i><b>3.2.1</b> What are Markers ? ü§∑üèªÔ∏è</a></li>
<li class="chapter" data-level="3.2.2" data-path="genomics.html"><a href="genomics.html#what-is-an-association-study"><i class="fa fa-check"></i><b>3.2.2</b> What is an Association Study ?</a></li>
<li class="chapter" data-level="3.2.3" data-path="genomics.html"><a href="genomics.html#microsatellites"><i class="fa fa-check"></i><b>3.2.3</b> Microsatellites üõ∞</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="systems-biology.html"><a href="systems-biology.html"><i class="fa fa-check"></i><b>4</b> Systems Biology</a><ul>
<li class="chapter" data-level="4.1" data-path="systems-biology.html"><a href="systems-biology.html#molecules-to-pathways"><i class="fa fa-check"></i><b>4.1</b> Molecules to Pathways</a><ul>
<li class="chapter" data-level="4.1.1" data-path="systems-biology.html"><a href="systems-biology.html#camp-pathway"><i class="fa fa-check"></i><b>4.1.1</b> cAMP Pathway</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html"><i class="fa fa-check"></i><b>5</b> Statistics in Medicine</a><ul>
<li class="chapter" data-level="5.1" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#samples-vs-population"><i class="fa fa-check"></i><b>5.1</b> Samples vs Population</a><ul>
<li class="chapter" data-level="5.1.1" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#simple-random-sampling"><i class="fa fa-check"></i><b>5.1.1</b> Simple Random Sampling</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#study-design"><i class="fa fa-check"></i><b>5.2</b> Study Design</a><ul>
<li class="chapter" data-level="5.2.1" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#prospective-cohort-studies"><i class="fa fa-check"></i><b>5.2.1</b> Prospective Cohort Studies</a></li>
<li class="chapter" data-level="5.2.2" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#case-control-studies"><i class="fa fa-check"></i><b>5.2.2</b> Case / Control Studies</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#data-types"><i class="fa fa-check"></i><b>5.3</b> Data Types</a><ul>
<li class="chapter" data-level="5.3.1" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#continuous-data"><i class="fa fa-check"></i><b>5.3.1</b> Continuous Data</a></li>
<li class="chapter" data-level="5.3.2" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#categorical-data"><i class="fa fa-check"></i><b>5.3.2</b> Categorical Data</a></li>
<li class="chapter" data-level="5.3.3" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#time-to-event-data"><i class="fa fa-check"></i><b>5.3.3</b> Time-to-Event Data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#linear-regression"><i class="fa fa-check"></i><b>5.4</b> Linear Regression</a><ul>
<li class="chapter" data-level="5.4.1" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#basic-terminology"><i class="fa fa-check"></i><b>5.4.1</b> Basic Terminology</a></li>
<li class="chapter" data-level="5.4.2" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#simple-linear-regression-with-a-binary-predictor"><i class="fa fa-check"></i><b>5.4.2</b> Simple Linear Regression with a Binary Predictor</a></li>
<li class="chapter" data-level="5.4.3" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#simple-linear-regression-with-a-categorical-predictor"><i class="fa fa-check"></i><b>5.4.3</b> Simple Linear Regression with a Categorical Predictor</a></li>
<li class="chapter" data-level="5.4.4" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#simple-linear-regression-with-a-continuous-predictor"><i class="fa fa-check"></i><b>5.4.4</b> Simple Linear Regression with a Continuous Predictor</a></li>
<li class="chapter" data-level="5.4.5" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#estimating-the-regression-equation"><i class="fa fa-check"></i><b>5.4.5</b> Estimating the Regression Equation</a></li>
<li class="chapter" data-level="5.4.6" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#measuring-the-strength-of-a-linear-association"><i class="fa fa-check"></i><b>5.4.6</b> Measuring the Strength of a Linear Association</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#logistic-regression"><i class="fa fa-check"></i><b>5.5</b> Logistic Regression</a><ul>
<li class="chapter" data-level="5.5.1" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#overview"><i class="fa fa-check"></i><b>5.5.1</b> Overview</a></li>
<li class="chapter" data-level="5.5.2" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#estimation"><i class="fa fa-check"></i><b>5.5.2</b> Estimation</a></li>
<li class="chapter" data-level="5.5.3" data-path="statistics-in-medicine.html"><a href="statistics-in-medicine.html#estimating-risk-and-risk-functions"><i class="fa fa-check"></i><b>5.5.3</b> Estimating Risk and Risk Functions</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Bioinformatics Notes üß¨</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="genomics" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Genomics üß¨</h1>
<div id="basics-of-genomics" class="section level2">
<h2><span class="header-section-number">3.1</span> Basics of Genomics üß¨</h2>
<div id="transcription" class="section level3">
<h3><span class="header-section-number">3.1.1</span> Transcription</h3>
<ul>
<li>Genetic Information can be:
<ul>
<li>Genomic (concerning the DNA)</li>
<li>Transcriptomic (concerning the transcriptome RNA)</li>
</ul></li>
</ul>
<p>Gene expression is the process by which a product such as a protein or RNA is created by a gene. Transcription is the process of producing the transcriptome from the genome. DNA contains all the information necessary to create new macromolecules such as proteins and beyond human beings. DNA encodes the data representing the blueprint of a particular person. The process by which a DNA is transcribed into corresponding RNA proceeds through a series of steps.</p>
<ol style="list-style-type: decimal">
<li><strong>Initiation</strong> ‚Üí DNA macromolecule opens and an enzyme (RNA polymerase) binds with the promoter of the template strand</li>
<li><strong>Elongation</strong> ‚Üí RNA polymerase processes each nucleotide on the DNA template strand to produce mRNA</li>
<li><strong>Termination</strong> ‚Üí Transcription stops either when meeting a ‚ÄúRho‚Äù protein factor or a loop at the end of the DNA strand</li>
<li><strong>Processing</strong> ‚Üí Newly created mRNA macromolecule is processed by removing the introns and slicing the exons together so that only regions coding for proteins are maintained.</li>
</ol>
<p>During this process, mRNA nucleotides are synthesized following a very strict process:</p>
<ul>
<li>DNA A ‚Üí mRNA U</li>
<li>DNA T ‚Üí mRNA A</li>
<li>DNA C ‚Üí mRNA G</li>
<li>DNA G ‚Üí mRNA C</li>
</ul>
</div>
<div id="translation" class="section level3">
<h3><span class="header-section-number">3.1.2</span> Translation</h3>
<p>Translation is the process of producing a protein from a messenger RNA. Proteins are made up of amino aicds. Each amino acid is coded by three nucleotides in the mRNA. The sequence of aminoacids composing a protein is synthesized by one by one and glued together as the translation progresses, in the order dictated by the strand. This process takes place in ribosomes, organelles located in the cytoplasm where the mRNA has been led after the transcription. A ribosoome reads the mRNA strand in groups of three bases to assemble the protein. There are 20 aminoacids in humans. A certain combination of 3 bases, also called a codon, always leads to the same aminoacid. There are 64 codons, out of which 61 code for aminoacids, plus 3 stop codons. Most aminoacids are therefore coded by several different codons</p>
</div>
</div>
<div id="genetic-markers" class="section level2">
<h2><span class="header-section-number">3.2</span> Genetic Markers üñä</h2>
<p>In this section, we‚Äôll learn about genetic markers and their use in association studies.</p>
<div id="what-are-markers-Ô∏è" class="section level3">
<h3><span class="header-section-number">3.2.1</span> What are Markers ? ü§∑üèªÔ∏è</h3>
<p>Genetic markers can be described as measurable variation (polymorphism) at the DNA level. They are chromosome regions where differences in nucleotide sequences occur between individuals of the same species. Various types of markers have been used to detect DNA variability, some of the most common ones are:</p>
<ul>
<li>Restriction fragment length polymorphism (<strong>RFLP</strong>)</li>
<li>Random amplification of polymorphic DNA (<strong>RAPD</strong>)</li>
<li>Amplified fragment length polymorphism (<strong>AFLP</strong>)</li>
<li>Single stranded conformation polymorphism (<strong>SSCP</strong>)</li>
<li>Copy number variation (<strong>CNV</strong>)</li>
<li>Microsatellites</li>
<li>Single nucleotide polymorphism (<strong>SNP</strong>)</li>
</ul>
<p>Although, <strong>microsatellites</strong> and <strong>SNP</strong> are the most commonly used markers.</p>
<hr />
<p><strong>NOTE:</strong> Their is a difference between a genetic marker and DNA variation. A marker is more related to the technique employed to measure the variation whilst DNA variation is obviously independent of our ability to measure it or not. When we talk about DNA variation, we should think more in terms of:</p>
<ul>
<li><strong>indels</strong> - insertions or deletions of various lengths</li>
<li><strong>VNTRs</strong> - variable number of tandem repeats (multiple copies of the same sequence of DNA, usually simple non-coding repeats)</li>
<li><strong>SNP</strong> - single nucleotide polymorphisms which are a single nucleotide mutations (base mutations).</li>
</ul>
<hr />
</div>
<div id="what-is-an-association-study" class="section level3">
<h3><span class="header-section-number">3.2.2</span> What is an Association Study ?</h3>
<p>The objective of an association study is to link a specific DNA region (and it‚Äôs underlying variation) with a trait of interest. For example in human research, many studies have been undertaken to identify genomic regions which confer greater resistance/susceptibility to diseases such as cancer, and ultimately, within these regions, identify the variants that lead to expression of the trait.</p>
<ul>
<li><p>Knowledge of such <em>functional</em> variations improves our understanding of disease mechanisms, suggests target for novel therapeutics and can be useful for risk prediction and prevention.</p></li>
<li><p>Identification of regions associated with a trait can also help select candidate regions for gene discovery - i.e, identify the actual genes and the functional mutations that confer the trait differences.</p></li>
</ul>
<hr />
<p><strong>NOTE that an association does not imply causality. A marker can be associated with a trait but it is not necessarily in any way related to it.</strong></p>
<p>A Chromosome is a physical structure of nucleotides connected to each other which only get broken up by recombination events. Hence, we can think of DNA sequences being transmitted from parent to offspring in <em>chunks</em>. The closer the marker is to the actual causative gene, the greater the chance that the same marker allele will be transmitted to the offspring with the same causative allelic variant. Thus, <strong>the marker and the gene are in linkage on the same DNA chunk, or formally <em>haplotype</em> </strong>. A <strong>haplotype</strong> is a chunk of DNA which has a combination of alleles that would not be observed by chance if each one was segregating independently. This leads us to the concept of <strong>linkage disequilibrium</strong> (LD): a non-random association of alleles. A linkage is like the actual physical connection of nucleotides on a DNA strand. Whereas, linkage disequilibrium is a statistical test of significance, but it‚Äôs not necessarily telling us anything about the actual physical proximity of what we‚Äôre testing.</p>
<hr />
</div>
<div id="microsatellites" class="section level3">
<h3><span class="header-section-number">3.2.3</span> Microsatellites üõ∞</h3>
<p>Microsatellites are a type of VNTR (variable number of tandem repeats). If the length of the repeating unit is <em>less than five base pairs</em>, the VNTR is referred to as a microsatellite. If the length of the repeating unit is <em>greater than five base pairs</em> the VNTR is called a minisatellite.</p>
<p><strong>Variation in microsatellites is measured through the number of repeats that an individual has of the recording unit.</strong> We can get a handle on microsatellites using PCR(<em>polymerase chain reaction</em>) methods. Basically, we amplify regions that have the repeating sequence and seperate them by virtue of their size. We can refer to these different sizes as an allele.</p>
<p>As discussed marker alleles are not necessarily causative of the trait. In the case of microsatellites it‚Äôs almost a given that they are not causative and that they are in non-coding regions. Microsatellites can have upto 10 to 30 different alleles (SNP‚Äôs generally have 2). A single microsatellite marker can be more informative than a SNP but they are generally more sparse with much lower LD between marker and the causal variant.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="basics.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="systems-biology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["genomic_notes.pdf", "genomic_notes.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
